packages/react-sdk/README.md

```diff
@@ -93,18 +108,20 @@ const optimizely = createInstance({
 })
 ```
 
-
 ## `<OptimizelyProvider>`
 
 Required at the root level. Leverages React’s `Context` API to allow access to the `ReactSDKClient` to components like `<OptimizelyFeature>` and `<OptimizelyExperiment>`.
 
 *props*
 * `optimizely : ReactSDKClient` created from `createInstance`
-* `user: { id: string; attributes?: { [key: string]: any } }` User info object - `id` and `attributes` will be passed to the SDK for every feature flag, A/B test, or `track` call
-* `timeout : Number` (optional) The amount of time for OptimizelyExperiment and OptimizelyFeature components to render `null` before resolving
+* `user: { id: string; attributes?: { [key: string]: any } } | Promise` User info object - `id` and `attributes` will be passed to the SDK for every feature flag, A/B test, or `track` call, or a `Promise` for the same kind of object
+* `timeout : Number` (optional) The amount of time for OptimizelyExperiment and OptimizelyFeature components to render `null` while waiting for the SDK instance to become ready, before resolving..
 * `isServerSide : Boolean` (optional) must pass `true` here for server side rendering
-
-### Load the datafile synchronously
+* `userId : String` (optional) Another way to provide user id. The `user` object prop takes precedence when both are provided.
```

Comment:
I thought these were mainly for backwards compatibility, Shouldn't we just simplify the outward facing API by encouraging users of the SDK to use `user` or `userId` & `userAttributes`?

Do you see a benefit of exposing both? Like consumers don't have to resolve the id at the same time as attributes?

Reply:
Yep we're on the same page, those are for backwards compatibility, but I wanted to mention all props. I would encourage to everyone use `user`.

Reply:
I would suggest removing `userAttributes` and `userId` then. Or perhaps mark them as deprecated, use `user` instead to push people to the right API

Reply:
Will mark as deprecated 👍